### PR TITLE
User profile 1

### PR DESCRIPTION
**Pull request by OjeikuA**
## Description
I completed the user profile. I still have to add credentials for logging into gcloud (I don't think uploading images will work on another computer because I logged in via OAuth on my computer). I also am able to change the password to Surekha's "rex' user. But otherwise I believe this Component will be quite operational when you check it out

## How to test
No tests for this yet

## Notes
I have a deployed MongoDB instance that we could use - if others would like to use it, I can send you the credentials

## Issue tracking
https://trello.com/c/SWvyRGYM

### CODE REVIEW

File: client/state/atoms/userProfile.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { atom } from 'recoil';
+
+const userProfileState = atom({
+  key: 'UserProfileState',
+  default: {
```
Comment (Laweeza) [inline client/state/atoms/userProfile.js:5]: looks good

File: package.json [hunk 2]
```diff
@@ -15,13 +15,17 @@
   },
   "homepage": "https://github.com/rpp30-boc-heartbleed/whimsical#readme",
   "dependencies": {
+    "@google-cloud/storage": "^5.16.1",
     "axios": "^0.24.0",
     "dotenv": "^10.0.0",
     "expo": "~43.0.2",
     "expo-status-bar": "~1.1.0",
-    "express": "^4.17.1",
```
Comment (Laweeza) [inline package.json:22]: looks like package-json version conflict has been resolved

File: server/controllers/userProfile.js [hunk 3]
```diff
@@ -1,13 +1,92 @@
...
+const storage = new Storage();
+
+const app = express();
+app.set('view engine', 'pug');
+
+// This middleware is available in Express v4.16.0 onwards
+app.use(express.json());
+
+// A bucket is a container for objects (files).
+const bucket = storage.bucket(process.env.GCLOUD_STORAGE_BUCKET);
```
Comment (Laweeza) [inline server/controllers/userProfile.js:26]: interested to see how this compares to S3 bucket

**Review by Laweeza — APPROVED** [review]
Lgtm so far!

**Review by surekhaw — APPROVED** [review]
looks great!  but would you mind letting us know what the new password is for rex? or change it back, please? thank you!

**Review by shabbyblue16 — APPROVED** [review]
Great work on the back end, component's looking solid too

**Pull request merged**